Mathlib/SetTheory/Surreal/Basic.lean

```diff
@@ -1,9 +1,10 @@
 /-
 Copyright (c) 2019 Mario Carneiro. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
-Authors: Mario Carneiro, Scott Morrison
+Authors: Mario Carneiro, Scott Morrison, Violeta Hernández Palacios, Junyan Xu, Theodore Hwa
 -/
 import Mathlib.Algebra.Order.Hom.Monoid
+import Mathlib.Logic.Hydra
```

Comment:
I'd suggest deferring any material that needs this import to a separate file.

Reply:
e.g. `Surreal.Multiplication`.

Reply:
The `Hydra` material is needed at the early parts of the multiplication proof so I don't see a way to do that without moving the whole multiplication proof to a separate file. Maybe that is a good idea anyway since the multiplication proof is quite complicated on its own. I will do that.